fastlane/Fastfile

```diff
@@ -418,7 +418,8 @@ platform :ios do
     en_lproj = 'WordPress/Resources/en.lproj'
     ios_generate_strings_file_from_code(
       paths: ['WordPress/', 'Pods/WordPress*/', 'Pods/WPMediaPicker/', 'WordPressShared/WordPressShared/', 'Pods/Gutenberg/'],
-      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/WordPressStatsWidgets/Views/Localization/LocalizedStringKey+extension.swift', '*/Secrets.swift'],
+      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/WordPressStatsWidgets/Views/Localization/AppLocalizedString.swift'],
+      routines: ['AppLocalizedString'],
```

Comment:
Figured there was no chance to have localized strings in the `Secrets.swift` file, and I'm actually not sure why it was excluded in the first place (back when it was in `localize.py` which is why I initially put it there in the previous PR), so figured I'd use the occasion to remove it (I checked and the lane still works without error nor any new key it shouldn't have)

Reply:
🤔 the only reason I can think of for ignoring it is to avoid `genstrings` accidentally printing out a secret that happened to match the localization regex but wasn't meant to. Maybe? I'm not even sure if that's something that could happen on the `genstrings` end, though. 🤷 

I agree that it should be 99.9% safe to remove it. 👍